02. Inputs and Outputs/README.md

```diff
@@ -0,0 +1,88 @@
+# Inputs and Outputs (Console)
+
+Lorem ipsum
+<-- rmb talk about escape characters, i believe most langs are almost the same for escape characters -->
```

Comment:
Just a side note, you forgot `!` after `<`, but I will edit that for you

Reply:
oof lol ok